subtests/docker_cli/events/test_event_parsing.py

```diff
@@ -0,0 +1,167 @@
+# -*- python -*-
+#
+from unittest2 import TestCase, main        # pylint: disable=unused-import
+import autotest  # pylint: disable=unused-import
+from dockertest.output import DockerTime
+import events
+
+
+class TestEventParsing(TestCase):
+    def test_1_09(self):
+        """
+        docker-1.9 format: container events
+        """
+        t = ["2016-04-06T09:53:33.265109190-04:00",
+             "2016-04-06T09:53:38.048694595-04:00",
+             "2016-04-06T09:53:41.016729639-04:00"]
+        cid = ("39b75e2aef85774dc545acc9998f0c44"
+               "d0af5f1baf32617e8bc5ed5f998cc558")
+        source = "registry.access.redhat.com/rhel7/rhel:latest"
+
+        event_log = ("{t[0]} {cid}: (from {source}) create\n"
+                     "{t[1]} {cid}: (from {source}) start\n"
+                     "{t[2]} {cid}: (from {source}) die\n".format(**locals()))
+        keys = ['datetime', 'identifier', 'operation', 'source']
+        expect = [
+            (cid, dict(zip(keys, [DockerTime(t[0]), cid, 'create', source]))),
+            (cid, dict(zip(keys, [DockerTime(t[1]), cid, 'start',  source]))),
+            (cid, dict(zip(keys, [DockerTime(t[2]), cid, 'die',    source])))]
+
+        actual = events.parse_events(event_log)
+        self.maxDiff = None
+        self.assertEqual(actual, expect)
+
+    def test_1_09_fqin(self):
+        """
+        docker-1.9 format: image events
+        """
+        t = ['2016-04-05T15:46:35.284845995-04:00',
+             '2016-04-05T15:46:35.344527102-04:00',
+             '2016-04-05T15:46:35.467081301-04:00']
+        fqins = ['docker.io/stackbrew/centos:7',
+                 'double_tag_c5dv:',
+                 'double_tag_c5dv:latest_fikfjiub']
+        actions = ['pull', 'tag', 'tag']
+        event_log = ("{t[0]} {fqins[0]}: {actions[0]}\n"
+                     "{t[1]} {fqins[1]}: {actions[1]}\n"
+                     "{t[2]} {fqins[2]}: {actions[2]}\n".format(**locals()))
+        expect = []
+        for i in range(3):
+            expect.append((fqins[i], {'datetime':   DockerTime(t[i]),
+                                      'identifier': fqins[i],
+                                      'operation':  actions[i],
+                                      'source':     None}))
```

Comment:
Yeah, you're right, it's much more readable this way.  Global disable for pep is fine then.